.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - commit-message:
+      prefix: "Changelog:All"
+    directory: /tests
+    package-ecosystem: pip
+    schedule:
+      interval: weekly
+
```

Comment:
```suggestion
  - commit-message:
      prefix: "Changelog:All"
    directory: /
    package-ecosystem: docker
    schedule:
      interval: weekly
  - commit-message:
      prefix: "Changelog:All"
    directory: /
    package-ecosystem: gitsubmodule
    schedule:
      interval: weekly

```

Reply:
Will "submodule" include the mender_test_containers submodule? I'm not sure how well that will work. We don't keep our test API perfectly stable, and the idea was to update that submodule only when needed.

Reply:
I don't really know if and how it will work (I wasn't even aware that the "new" dependabot supports submodules) - it might be a good way to be reminded of the differences between the repos though... or do you think a successful CI run & build in this repo could cause trouble for the demo-artifact in other repos?

Reply:
If it passes, we can merge it. However, if it fails, the expectation is not that we will fix it. Rather we should keep it stable at the version which works, until we actually *need* to upgrade. So in that case updates from dependabot may pile up without ever getting merged.

But yes, you're right that it's a good way to be reminded, and mender-test-containers is not updated that often. We can just go ahead and try. If we do see many PRs being opened that we cannot merge, we can reevaluate.

Reply:
I agree: let's try and possibly remove if such situation arrives.